<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ASHWIN_RAMTEKE" failures="1" tests="7" name="Test_NG" time="58.922" errors="0" timestamp="2024-03-13T12:04:27 IST">
  <testcase classname="com.crm.qa.testcases.Manager_AddCustomer_test" name="verifyAddCustomerFunctionality" time="3.283"/>
  <testcase classname="com.crm.qa.testcases.Manager_AddCustomerInvalid_test" name="verifyAddCustomerFunctionality" time="2.414">
    <failure type="java.lang.AssertionError" message="Error message is not displayed for invalid data expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Error message is not displayed for invalid data expected [true] but found [false]
at com.crm.qa.testcases.Manager_AddCustomerInvalid_test.verifyAddCustomerFunctionality(Manager_AddCustomerInvalid_test.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- verifyAddCustomerFunctionality -->
  <testcase classname="com.crm.qa.testcases.Manager_Customers_test" name="OpenAccountCustomerTest" time="5.02"/>
  <testcase classname="com.crm.qa.testcases.Manager_OpenAccount_test" name="OpenAccountCustomerTest" time="5.312"/>
  <testcase classname="com.crm.qa.testcases.Customer_Transaction_test" name="DepositCustomerTest" time="8.838"/>
  <testcase classname="com.crm.qa.testcases.Customer_Withdarw_test" name="DepositCustomerTest" time="6.817"/>
  <testcase classname="com.crm.qa.testcases.Customer_Deposit_test" name="DepositCustomerTest" time="5.897"/>
</testsuite> <!-- Test_NG -->
